components/Card.js

```diff
@@ -1,7 +1,48 @@
 import React from "react";
+import Card from "react-bootstrap/Card";
 
-const Card = () => {
-	return <div>Card</div>;
-};
+class Block extends React.Component {
```

Comment:
Use functional arrow components

Reply:
so change it back to const Card = () =>?

Reply:
yup!